api/interactives.go

```diff
@@ -194,8 +194,30 @@ func (api *API) UpdateInteractiveHandler(w http.ResponseWriter, req *http.Reques
 
 }
 
-func (api *API) ListInteractivessHandler(w http.ResponseWriter, req *http.Request) {
+func (api *API) ListInteractivesHandler(w http.ResponseWriter, req *http.Request, limit int, offset int) (interface{}, int, error) {
```

Comment:
i think better to abstract limt & offset into struct?

Reply:
something like https://github.com/ONSdigital/dp-census-alpha-search-api/blob/9cc640957c2096bf3d95547a4be2fd8cb1d19a41/models/pagination.go#L11 - bit cleaner imo but upto you :) prob not with that name though

Reply:
Agreed and it be more easy on the eyes too, point noted .. will do that in a future checkin, thanks